addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -271,6 +271,12 @@ def _get_invoice_line_vals(self, line, taxes_vals):
         vals['currency_dp'] = 2
         vals['price_vals']['currency_dp'] = 2
 
+        if line.currency_id.compare_amounts(vals['price_vals']['price_amount'], 0) == -1:
+            # We can't have negative unit prices, so we invert the signs of
+            # the unit price and quantity, resulting in the same amount in the end
+            vals['price_vals']['price_amount'] *= -1
+            vals['invoiced_quantity'] *= -1
```

Comment:
Pay attention during the fw-port that the key name changes in 17.0: https://github.com/odoo/odoo/blob/17.0/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py#L380